doc/encryption.md

```diff
@@ -146,7 +146,7 @@ The exception has the form:
      Caused by: java.lang.RuntimeException: Unexpected type: 5
             at com.cloudant.sync.datastore.BasicDatastore.getFullRevisionFromCurrentCursor(BasicDatastore.java:1709)
             at com.cloudant.sync.datastore.BasicDatastore.getDocumentInQueue(BasicDatastore.java:284)
-            at com.cloudant.sync.datastore.BasicDatastore.createDocument(BasicDatastore.java:681)
+            at com.cloudant.sync.datastore.BasicDatastore.createJsonDocument(BasicDatastore.java:681)
```

Comment:
This would be the stacktrace from this commit on though, I think? I guess I could fix up the line numbers, but I guess you're not being super-serious?